### ISSUE TITLE
Cannot run imagenet_pretrained_vgg example

### ISSUE THREAD

**Issue by RXZ2020** [google/objax#160]
I was trying to run https://github.com/google/objax/blob/master/examples/image_classification/imagenet_pretrained_vgg.py

But received an error:

> ---------------------------------------------------------------------------
> FileNotFoundError                         Traceback (most recent call last)
> <ipython-input-2-cf1dcb75d1ba> in <module>()
>      38 
>      39 # Load model with pretrained weights and make a prediction.
> ---> 40 model = vgg.VGG19(pretrained=True)
>      41 logit = model(img)
>      42 prob = objax.functional.softmax(logit)[0]
> 
> /usr/local/lib/python3.6/dist-packages/objax/zoo/vgg.py in __init__(self, pretrained)
>      55         if not os.path.exists(_VGG19_NPY):
>      56             raise FileNotFoundError(
> ---> 57                 'You must download vgg19.npy from %s and save it to %s' % (_VGG19_URL, _VGG19_NPY))
>      58         if not os.path.exists(_SYNSET_PATH):
>      59             request.urlretrieve(_SYNSET_URL, _SYNSET_PATH)
> 
> FileNotFoundError: You must download vgg19.npy from https://github.com/machrisaa/tensorflow-vgg and save it to ./objax/zoo/pretrained/vgg19.npy

**Comment by aterzis-google** [comment]
@RXZ2020 please download vgg19.npy from the URL mentioned in the error message and save it to ./objax/zoo/pretrained/vgg19.npy

**Comment by david-berthelot** [comment]
You know I've come to realize that's not really practical. If you install Objax with `pip install`, the path where to save may not be obvious. 

I've seen other frameworks create their own area say `$HOME/.objax`, maybe it something we should think about.

**Comment by RXZ2020** [comment]
> @RXZ2020 please download vgg19.npy from the URL mentioned in the error message and save it to ./objax/zoo/pretrained/vgg19.npy

Hi, how do you do this if I was running it on Colab? Do I create an objax folder in Google drive or elsewhere?

**Comment by david-berthelot** [comment]
Yes, good point. We should make Objax download it for you otherwise it won't be useful in a Colab.

**Comment by aterzis-google** [comment]
If installing Objax with `pip install`, then need to copy under site-packages/objax/zoo/pretrained. 

Not sure what the copyright for pretrained VGG is and whether we can include it in the objax package.
